src/ElectricPokemon.java

```diff
@@ -0,0 +1,112 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+
+    public static final String type = "electric";
+    private static final List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super("electric", name, level, hp, food, sound);
+    }
+
+    public void thunderPunch(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " hit " + enemy.getName() + " with ThunderPunch");
+        switch (enemy.getType()) {
+            case "water":
```

Comment:
Deze switch statements zijn 4 keer bijna hetzelfde. Het enige verschil zijn de punten. Je zou dit dus goed in een helper methode kunnen zetten waarin je de punten variabel maakt.